vendor/github.com/pelletier/go-toml/example-crlf.toml

```diff
@@ -1,30 +1,30 @@
-# This is a TOML document. Boom.
-
-title = "TOML Example"
-
-[owner]
-name = "Tom Preston-Werner"
-organization = "GitHub"
-bio = "GitHub Cofounder & CEO\nLikes tater tots and beer."
-dob = 1979-05-27T07:32:00Z # First class dates? Why not?
-
-[database]
-server = "192.168.1.1"
-ports = [ 8001, 8001, 8002 ]
-connection_max = 5000
-enabled = true
-
-[servers]
-
-  # You can indent as you please. Tabs or spaces. TOML don't care.
-  [servers.alpha]
-  ip = "10.0.0.1"
-  dc = "eqdc10"
-
-  [servers.beta]
-  ip = "10.0.0.2"
-  dc = "eqdc10"
-
-[clients]
-data = [ ["gamma", "delta"], [1, 2] ] # just an update to make sure parsers support it
+# This is a TOML document. Boom.
```

Comment:
This may become a fun one; I was wondering why this changed, but looks like this is a "CRLF" test file. Re-vendoring on my local machine seems to revert this back to the original.

Perhaps we need to add a special case in our `.gitattributes` if that's gonna cause issues 🤔

Reply:
I'll check what CI says, otherwise I do a quick push to your branch to fix it

Reply:
Yup, looks like `vendor` check didn't like it (for some reason it didn't error out, but got stuck; not sure why that was)

```
 rm -f /go/src/github.com/docker/docker/go.mod
+ /go/src/github.com/docker/docker/hack/with-go-mod.sh go mod vendor -modfile vendor.mod
+ tee /go/src/github.com/docker/docker/go.mod
module github.com/docker/docker

go 1.18
+ trap 'rm -f "${ROOTDIR}/go.mod"' EXIT
+ GO111MODULE=on
+ go mod vendor -modfile vendor.mod
+ rm -f /go/src/github.com/docker/docker/go.mod
FAIL: Vendoring was not performed correctly!

The following files changed during re-vendor:

M	vendor/github.com/pelletier/go-toml/example-crlf.toml

Please revendor with hack/vendor.sh

diff --git a/vendor/github.com/pelletier/go-toml/example-crlf.toml b/vendor/g
ithub.com/pelletier/go-toml/example-crlf.toml
index f45bf88b8f..780d9c68f2 100644
--- a/vendor/github.com/pelletier/go-toml/example-crlf.toml
+++ b/vendor/github.com/pelletier/go-toml/example-crlf.toml
@@ -1,30 +1,30 @@
-# This is a TOML document. Boom.
-
-title = "TOML Example"
-
-[owner]
-name = "Tom Preston-Werner"
-organization = "GitHub"
-bio = "GitHub Cofounder & CEO\nLikes tater tots and beer."
-dob = 1979-05-27T07:32:00Z # First class dates? Why not?
-
-[database]
-server = "192.168.1.1"
-ports = [ 8001, 8001, 8002 ]
-connection_max = 5000
-enabled = true
-
-[servers]
```